managed/services/agents/node.go

```diff
@@ -30,7 +30,9 @@ import (
 // The node exporter prior 2.28 use exporter_shared and gets basic auth config from env.
 // Starting with pmm 2.28, the exporter uses Prometheus Web Toolkit and needs a config file
 // with the basic auth users.
-var v2_27_99 = version.MustParse("2.27.99")
+var (
+	v2_28_00 = version.MustParse("2.28.0-0")
```

Comment:
Btw do you think we should eventually move this to features for consistency?

Reply:
I think we need to drop it eventually :)